ethcore/src/client/client.rs

```diff
@@ -2739,12 +2740,13 @@ fn transaction_receipt(
 
 /// Queue some items to be processed by IO client.
 struct IoChannelQueue {
-	currently_queued: Arc<AtomicUsize>,
-	limit: usize,
+	currently_queued: Arc<AtomicI64>,
```

Comment:
Please add docs why we're using signed integers here, so it won't get accidentally reverted in the future.

Reply:
I added documentation to the `currently_queued` field to clarify why signed integers are being used here.